.config/webpack/webpack.config.ts

```diff
@@ -12,188 +13,201 @@ import path from 'path';
+        lintDirtyModulesOnly: Boolean(env.development), // don't lint on start, only lint changed files
+      }),
+      ...(env.development ? [new LiveReloadPlugin()] : []),
     ],
-  },
 
-  output: {
-    clean: {
-      keep: /gpx_.*/,
-    },
-    filename: '[name].js',
-    library: {
-      type: 'amd',
+    resolve: {
+      extensions: ['.js', '.jsx', '.ts', '.tsx'],
+      // handle resolving "rootDir" paths
+      modules: [path.resolve(process.cwd(), 'src'), 'node_modules'],
+      unsafeCache: true,
     },
-    path: path.resolve(process.cwd(), DIST_DIR),
-    publicPath: '/',
-  },
-
-  plugins: [
-    new CopyWebpackPlugin({
-      patterns: [
-        // If src/README.md exists use it; otherwise the root README
-        // To `compiler.options.output`
-        { from: hasReadme() ? 'README.md' : '../README.md', to: '.', force: true },
-        { from: 'plugin.json', to: '.' },
-        { from: '../LICENSE', to: '.' },
-        { from: '../CHANGELOG.md', to: '.', force: true },
-        { from: '**/*.json', to: '.' }, // TODO<Add an error for checking the basic structure of the repo>
-        { from: '**/*.svg', to: '.', noErrorOnMissing: true }, // Optional
-        { from: '**/*.png', to: '.', noErrorOnMissing: true }, // Optional
-        { from: '**/*.html', to: '.', noErrorOnMissing: true }, // Optional
-        { from: 'img/**/*', to: '.', noErrorOnMissing: true }, // Optional
-        { from: 'libs/**/*', to: '.', noErrorOnMissing: true }, // Optional
-        { from: 'static/**/*', to: '.', noErrorOnMissing: true }, // Optional
-      ],
-    }),
-    // Replace certain template-variables in the README and plugin.json
-    new ReplaceInFileWebpackPlugin([
-      {
-        dir: DIST_DIR,
-        files: ['plugin.json', 'README.md'],
-        rules: [
-          {
-            search: /\%VERSION\%/g,
-            replace: getPackageJson().version,
-          },
-          {
-            search: /\%TODAY\%/g,
-            replace: new Date().toISOString().substring(0, 10),
-          },
-          {
-            search: /\%PLUGIN_ID\%/g,
-            replace: getPluginId(),
-          },
-        ],
-      },
-    ]),
-    new ForkTsCheckerWebpackPlugin({
-      async: Boolean(env.development),
-      issue: {
-        include: [{ file: '**/*.{ts,tsx}' }],
-      },
-      typescript: { configFile: path.join(process.cwd(), 'tsconfig.json') },
-    }),
-    new ESLintPlugin({
-      extensions: ['.ts', '.tsx'],
-      lintDirtyModulesOnly: Boolean(env.development), // don't lint on start, only lint changed files
-    }),
-    ...(env.development ? [new LiveReloadPlugin()] : []),
-  ],
-
-  resolve: {
-    extensions: ['.js', '.jsx', '.ts', '.tsx'],
-    // handle resolving "rootDir" paths
-    modules: [path.resolve(process.cwd(), 'src'), 'node_modules'],
-    unsafeCache: true,
-  },
-});
+  }
+
+  if(isWSL()) {
+    baseConfig.watchOptions = {
+      poll: 3000,
+      ignored: /node_modules/,
+    }}
```

Comment:
Should the `}` be propped to the next line?

Reply:
I think the `eslint` config for the `create-plugin` tool may be slightly different to our config so I haven't touched these files